tests/spread/rockcraft/plugin-ant/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: Maven plugin tests
+execute: |
+  SCENARIO="plugin-ant"
+  ROCK_FILE="${SCENARIO}_0.1_amd64.rock"
+  IMAGE="${SCENARIO}:0.1"
+
+  # Build the rock & load it into docker
+  run_rockcraft pack
+  echo expecting ${ROCK_FILE}
+  test -f ${ROCK_FILE}
+  rockcraft.skopeo copy --insecure-policy  oci-archive:${ROCK_FILE} docker-daemon:${IMAGE}
+  rm ${ROCK_FILE}
+
+  # Run the packaged project
+  docker run --rm $IMAGE exec /usr/bin/java -jar /jar/HelloWorld.jar | MATCH "hello world"
```

Comment:
please also run with `/bin/java` - it should exist because of the base-files slice, and it's important to catch future regressions